src/JSURI.js

```diff
@@ -34,7 +34,7 @@ function _decodeURIComponent(fail, succeed, input) {
   }
 }
 
-exports._decodeURIComponent = _decodeURIComponent;
```

Comment:
It seems odd that only this line was changed. What about the other `exports`?

Reply:
I suspect it didn't handle this correctly. Perhaps because the other exports are named function (all incorrectly named `encode` as it happens!)